tests/mir-opt/instrument_coverage.bar.InstrumentCoverage.diff

```diff
@@ -5,7 +5,7 @@
       let mut _0: bool;
   
       bb0: {
-+         Coverage::Counter(0) for [/the/src/instrument_coverage.rs:20:1 - 22:2];
```

Comment:
You're probably right. I hadn't bothered because I mostly rely on `coverage-map` tests and haven't needed that information in MIR dumps (yet), but it will probably be a lifesaver someday.